UploadDaemon_Test/SymbolAnalysis/GlobPatternListTest.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using NUnit.Framework;
-using Common;
 using System.Collections.Generic;
+using UploadDaemon.Utils;
 
 [TestFixture]
 public class GlobPatternListTest
```

Comment:
Move to namespace `UploadDaemon.Utils` or `UploadDaemon.SymbolAnalysis` depending on the decision where  `GlobPatternList.cs` should be located.